packages/treasury/src/liquidateMinimum.js

```diff
@@ -51,21 +45,17 @@ export async function start(zcf) {
         debtorSeat,
       );
 
-      // if swapOut failed for insufficient funds, we'll sell it all
-      async function onSwapOutFail(error) {
-        const strategy = makeDefaultLiquidationStrategy(autoswap);
+      // if swapOut failed to make the trade, we'll sell it all
+      async function onSwapOutNoTrade() {
+        const strategy = makeDefaultLiquidationStrategy(amm);
         trace(`onSwapoutFail`);
-        assert(
-          error.message.match(AutoswapInsufficientMsg),
-          `unable to liquidate: ${error}`,
-        );
 
         const {
           deposited: sellAllDeposited,
           userSeatPromise: sellAllSeat,
         } = await offerTo(
           zcf,
-          strategy.makeInvitation(),
+          strategy.makeInvitation(runDebt),
```

Comment:
`runDebt` doesn't seem to be used in the default strategy, but I see it in the `LiquidationStrategy` type declaration; I suppose `lint:types` prompted this change.